### ISSUE TITLE
Update CHANGES.rst file

### ISSUE THREAD

**Issue by petiay** [BEAST-Fitting/beast#681]
[CHANGES.rst](https://github.com/petiay/beast/blob/master/CHANGES.rst) can be updated with the BEAST v2.0 release info.

**Comment by karllark** [comment]
Good point.  We should be updating this in PRs that make substantial changes.  But at this point we don't and I update it when I make a formal release.  As 2.0 has not been released...

It would be good to update this file based on the merged PRs since the last release.  And then start reminding people to update it when they do a substantial PR.

**Comment by petiay** [comment]
I can try to compile the main changes I see from the merged PRs and maybe we can briefly discuss at the next Hack Day.

**Comment by karllark** [comment]
Awesome.

**Comment by karllark** [comment]
Done in recent release.
